standard/sections/clause_7_normative_text.adoc

```diff
@@ -196,12 +196,14 @@ include::../recommendations/core/PER_geometry.adoc[]
 
 ==== Properties / Publication Time
 
-The ``pubtime`` property identifies the date/time of when the origin notification was posted/published, in RFC3339 format, in
-the UTC timezone (``Z``). The publication date/time is critical for subscribers to prevent message loss in
-providing awareness of how far behind the publisher they may be).  ``pubtime`` is also valuable for change
-detection as part of updates and deletion notifications.  Ensuring ``pubtime`` is properly managed for updates
-and deletions is important for data and metadata download workflows (an out of date ``pubtime`` can lead
-to errors for clients when managing updates or deletions in their local copies of data).
+The ``pubtime`` property identifies the date/time of when the notification was first posted or published. The date/time is encoded in RFC3339 format with the UTC timezone (``Z``). 
+//FIX?
+The publication date/time is critical for subscribers to prevent message loss in providing awareness of how far behind the publisher they may be.  
+
+The ``pubtime`` property is also valuable for change detection as part of updates and deletion notifications.  
+
+//FIX?
+Ensuring ``pubtime`` is properly managed for updates and deletions is important for data and metadata download workflows (an out of date ``pubtime`` can lead to errors for clients when managing updates or deletions in their local copies of data).
```

Comment:
Ensuring ``pubtime`` is properly managed for updates and deletions is important for data and metadata download workflows.  For example, an out of date ``pubtime`` can lead to errors for clients when managing updates or deletions in their local copies of data.